src/main/java/it/gov/pagopa/payhub/auth/repository/ClientRepositoryExtImpl.java

```diff
@@ -0,0 +1,27 @@
+package it.gov.pagopa.payhub.auth.repository;
+
+import it.gov.pagopa.payhub.auth.model.Client;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+
+public class ClientRepositoryExtImpl implements ClientRepositoryExt {
```

Comment:
why are you introducing an extension for a simple delete query?
See:
https://docs.spring.io/spring-data/data-mongo/docs/1.5.0.RELEASE/reference/html/mongo.repositories.html
paragraph "Example 5.7. Delete…By Query"